package/src/utils/utils.ts

```diff
@@ -512,3 +513,16 @@ export const emojiRegex =
 
 export const urlRegex =
   /(?:\s|^)((?:https?:\/\/)?(?:[a-z0-9-]+(?:\.[a-z0-9-]+)+)(?::[0-9]+)?(?:\/(?:[^\s]+)?)?)/g;
+
+export const mapMessages = <
```

Comment:
`mapMessages` is such a generic naming. It doesn't document what this function does well. 

The returned string is used as a dependency for the useMemo. This must be clear. Add this is a documentation to the function. 

And also please think of a better name for the function. `mapMessages` => `getUpdatedMessages`, or something similar

Reply:
✅